event/listener.php

```diff
@@ -73,6 +74,23 @@ public function init_collapsible_categories()
 	}
 
 	/**
+	 * Load common language files during user setup
```

Comment:
For better organization, the functions should appear in the class in the same order that they are called in getSubscribedEvents.